### PR TITLE
Api 84 api tokens with prefix

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

File: packages/common/test/utils/token.utils.spec.ts [hunk 1]
```diff
@@ -3,20 +3,37 @@ import { TokenUtils } from "../../src/utils/token.utils";
...
     });
   });
 
   describe('isCollection', () => {
     it('Check isCollection function', () => {
       expect(TokenUtils.isCollection('MOS-b9b4b2')).toBeTruthy();
-      expect(TokenUtils.isCollection('MOS-b9b4b2-455c74')).toBeFalsy();
+      expect(TokenUtils.isCollection('sov-MOS-b9b4b2')).toBeTruthy();
+      expect(TokenUtils.isCollection('SOV-MOS-b9b4b2')).toBeFalsy();
+      expect(TokenUtils.isCollection('sov-MOS-b9b4b2-455c74')).toBeFalsy();
```
Comment (cfaur09) [inline packages/common/test/utils/token.utils.spec.ts:17]: * you can add Incomplete identifier scenarios like: ('sov-') to be Falsy

File: packages/common/test/utils/token.utils.spec.ts [hunk 2]
```diff
@@ -3,20 +3,37 @@ import { TokenUtils } from "../../src/utils/token.utils";
...
+    });
+  });
+
+  describe('isSovereignIdentifier', () => {
+    it('Check isSovereignIdentifier function', () => {
+      expect(TokenUtils.isSovereignIdentifier('sov-MOS-b9b4b2-947a3912')).toBeTruthy();
+      expect(TokenUtils.isSovereignIdentifier('sov-MOS-b9b4b2')).toBeTruthy();
+      expect(TokenUtils.isSovereignIdentifier('SOV-MOS-b9b4b2-947a3912')).toBeFalsy();
+      expect(TokenUtils.isSovereignIdentifier('SOV-MOS-b9b4b2')).toBeFalsy();
+      expect(TokenUtils.isSovereignIdentifier('MOS-b9b4b2-947a3912')).toBeFalsy();
```
Comment (cfaur09) [inline packages/common/test/utils/token.utils.spec.ts:37]: * also you can add test for empty string -> toBeFalsy
* also you can add test for Incomplete identifier -> toBeFalsy

File: packages/common/test/utils/token.utils.spec.ts (hunk 1 above)
Comment (bogdan-rosianu) [reply packages/common/test/utils/token.utils.spec.ts:22]: added

File: packages/common/test/utils/token.utils.spec.ts (hunk 2 above)
Comment (bogdan-rosianu) [reply packages/common/test/utils/token.utils.spec.ts:47]: added

**Review by cfaur09 — DISMISSED** [review]

**Review by stefangutica — DISMISSED** [review]

**Review by cfaur09 — DISMISSED** [review]

**Review by stefangutica — DISMISSED** [review]
